.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 .vscode
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.txt*
```

Comment:
Why `*.txt*` instead of `*.txt`?

Reply:
rotating file handler adds a number to the end of the file, not before the extension. I'll take another look. An earlier commit had me doing rotations by hand and that was stupid

Reply:
Opened #24 to see if we can get the number before the extension

Reply:
> rotating file handler adds a number to the end of the file, not before the extension

That makes sense and saw it in the docs, but misread it and thought it was name.num.ext instead of name.ext.num.